### PR TITLE
Docs: Pull request template

### PR DESCRIPTION
**Pull request by nzakas**
We've had quite a few pull requests opened recently without issues, so I'd like to see if adding a pull request template will help minimize that. This PR adds a simple template with just a few questions.

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+Please answer all questions below.
+-->
+
+**What issue does this pull request address?**
+
+
+**What changes did you make? (Give an overview)**
+
+
+**Is there anything you'd like to reviewers to focus on?**
```
Comment (platinumazure) [inline .github/PULL_REQUEST_TEMPLATE.md]: s/like to reviewers/like reviewers (or like for reviewers)

**Comment by platinumazure** [comment]
Small typo, otherwise LGTM

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (alberto) [inline .github/PULL_REQUEST_TEMPLATE.md]: Not only we require an issue, but it must also be accepted first. We don't want people to open an issue and submit a PR immediately.

Comment (nzakas) [inline .github/PULL_REQUEST_TEMPLATE.md]: @alberto how would you suggest updating what I have here?

Comment (alberto) [inline .github/PULL_REQUEST_TEMPLATE.md]: Specifically, **all pull requests containing code require an accepted issue** (documentation-only pull requests do not require an issue). If this pull requests contain code, and there isn't yet an issue explaining why you're submitting this pull request, please stop and open a new issue first. If the issue is not accepted yet, please wait.

**Comment by nzakas** [comment]
Updated.

**Comment by alberto** [comment]
Sorry, I missed this the first time:

"contain**s** code"

**Comment by scriptdaemon** [comment]
If this pull requests -> If this pull request

You can also remove the immediately following comma, as it is not a list of three or more items nor is it separating more than one independent clause.

**Comment by nzakas** [comment]
Thanks all, fixed.

**Pull request merged**